docs/core-library/h3indexing.md

```diff
@@ -76,7 +89,7 @@ The layout of an **H3Index** is shown below in table form. The interpretation of
   <th>0x30</th>
   <td>Reserved</td>
   <td colspan="4">Mode</td>
-  <td colspan="3">Reserved/edge</td>
+  <td colspan="3">Reserved</td>
```

Comment:
Technically this shouldn't be changed, or maybe changed to `unused/edge/vertex` to indicate that its value differs based on the mode. Reserved implies "not used at all by the specification".

Reply:
How about "mode-dependent"?